### DISCUSSION TITLE
do tournaments offer fair play?

### DISCUSSION THREAD

**Discussion by NishchayKQ** [efroemling/ballistica#455] [Q&A]
Are tournaments hack-proof as of now ? One thing the game already does is it invalidates scores of peeps who have plugins in mods folder
But does it prevent rooted devices from modification of game files ? I do know some people do mod their way up to leaderboards (having more achievements than there r; for example game 1000/41)

**Comment by TheMikirog** [comment]
There's several things worth noting:

Some tournament challenges can be cheesed for a much higher score, like in Meteor Shower how you can jump on top of the tiny ledge to avoid most of the bombs. The only thing keeping you from staying there forever would be the fact that you're a ball that slowly lowers. **Doing this does not require mods**.

Inflated achievements though seem to be a relic of the past, where you could create custom achievements in Python and these would get registered into your account without any security checks. You don't get a ticket reward for them anymore and they might not even be registered in the achievement list anymore afaik. If you did it before the fix, you'd have the inflated achievements progress.

I do not participate in tournaments, so take these findings with a grain of salt, especially the second one.

**Comment by NishchayKQ** [comment]
> Some tournament challenges can be cheesed for a much higher score, like in Meteor Shower how you can jump on top of the tiny ledge to avoid most of the bombs. The only thing keeping you from staying there forever would be the fact that you're a ball that slowly lowers. **Doing this does not require mods**.

1. This is really popular approach but to actually have a good score with this you have to play multiple times (to the point that its simply not worth it to win with this method)
the only bonus to the above approach is that you can get extremely lucky and make an absurdly good score with this method (so a nerf would be appreciated here...but i have observed that this approach is much less viable than it was in past ).
I personally don't use this and have managed to win quite a few without having to camp ledge
2.  oh! really happy it got patched!
